README.md

```diff
@@ -1000,17 +832,15 @@ c.store === store; // true
 Calling the `get` and `set` functions is equivalent to accessing the value on the instance.
 
 ```js
-const FIELD_ACCESS = Symbol();
-
 function exposeField(value, context) {
-  context.setMetadata(FIELD_ACCESS, context.access);
+  context.addMetadata(context.access);
 }
 
 class C {
   @exposeField #x = 1;
 
   updateX() {
-    let { get, set } = this[Symbol.metadata][FIELD_ACCESS].private[0];
+    let { get, set } = this.constructor[Symbol.metadata][0];
```

Comment:
```suggestion
    let { get, set } = C[Symbol.metadata][0];
```
probably better for examples not to use the unreliable `.constructor`

Reply:
Updated